src/connectors/clear-refinements/connectClearRefinements.ts

```diff
@@ -7,73 +7,83 @@ import {
- */
-
-/**
- * **ClearRefinements** connector provides the logic to build a custom widget that will give the user
- * the ability to reset the search state.
- *
- * This connector provides a `refine` function to remove the current refined facets.
- *
- * The behaviour of this function can be changed with widget options. If `clearsQuery`
- * is set to `true`, `refine` will also clear the query and `excludedAttributes` can
- * prevent certain attributes from being cleared.
- *
- * @type {Connector}
- * @param {function(ClearRefinementsRenderingOptions, boolean)} renderFn Rendering function for the custom **ClearRefinements** widget.
- * @param {function} unmountFn Unmount function called when the widget is disposed.
- * @return {function(CustomClearRefinementsWidgetOptions)} Re-usable widget factory for a custom **ClearRefinements** widget.
- * @example
- * // custom `renderFn` to render the custom ClearRefinements widget
- * function renderFn(ClearRefinementsRenderingOptions, isFirstRendering) {
- *   var containerNode = ClearRefinementsRenderingOptions.widgetParams.containerNode;
- *   if (isFirstRendering) {
- *     var markup = $('<button id="custom-clear-all">Clear All</button>');
- *     containerNode.append(markup);
- *
- *     markup.on('click', function(event) {
- *       event.preventDefault();
- *       ClearRefinementsRenderingOptions.refine();
- *     })
- *   }
- *
- *   var clearRefinementsCTA = containerNode.find('#custom-clear-all');
- *   clearRefinementsCTA.attr('disabled', !ClearRefinementsRenderingOptions.hasRefinements)
- * };
- *
- * // connect `renderFn` to ClearRefinements logic
- * var customClearRefinementsWidget = instantsearch.connectors.connectClearRefinements(renderFn);
- *
- * // mount widget on the page
- * search.addWidgets([
- *   customClearRefinementsWidget({
- *     containerNode: $('#custom-clear-all-container'),
- *   })
- * ]);
- */
-export default function connectClearRefinements(renderFn, unmountFn = noop) {
+export type ClearRefinementsConnectorParamsItem = {
+  /**
+   * Label to display in the option.
+   */
+  label: string;
+
+  /**
+   * Number of hits to display per page.
+   */
+  value: number;
+
+  /**
+   * The default hits per page on first search.
+   *
+   * @default false
+   */
+  default?: boolean;
+};
+
+export type ClearRefinementsConnectorParams = {
+  /**
+   * The attributes to include in the refinements to clear (all by default). Cannot be used with `excludedAttributes`.
+   */
+  includedAttributes?: string[];
+
+  /**
+   * The attributes to exclude from the refinements to clear. Cannot be used with `includedAttributes`.
+   */
+  excludedAttributes?: string[];
+
+  /**
+   * Function to transform the items passed to the templates.
+   */
+  transformItems?: TransformItems<ClearRefinementsConnectorParamsItem>;
+};
+
+export type ClearRefinementsRendererOptions = {
+  /**
+   * Triggers the clear of all the currently refined values.
+   */
+  refine: () => void;
```

Comment:
Can you confirm that there is no param here?

Reply:
Yep

https://github.com/algolia/instantsearch.js/blob/bae6ed2875b03a8a43c576085a19f2ceb43ba1f2/src/connectors/clear-refinements/connectClearRefinements.js#L130-L141